Project.toml

```diff
@@ -0,0 +1,32 @@
+name = "Blink"
+uuid = "ad839575-38b3-5650-b840-f874b8c74a25"
+authors = []
+version = "0.10.2"
```

Comment:
Might make sense to directly jump to `0.11`, considering the rather big changes in regard to WebIO.